docs/source/ko/_toctree.yml

```diff
@@ -655,6 +655,11 @@
       - local: in_translation
         title: (번역중) XLSR-Wav2Vec2
       title: (번역중) 오디오 모델
+      - isExpanded: false
+      sections:
+      - local: model_doc/timesformer.md
+        title: TimeSFormer
```

Comment:
I think your alignment may be off here. Should look like:

```yml
- isExpanded: false
  sections:
  - local: model_doc/timesformer
    title: TimeSFormer
```

Reply:
Thank you! I might have missed that. 😅

Reply:
Oops looks like the toctree still isn't properly aligned! The line [here](https://github.com/mreraser/transformers/blob/9ad85130c6e2c3636642418fca7c80d9aa339035/docs/source/ko/_toctree.yml#L658) should be aligned with the one [here](https://github.com/mreraser/transformers/blob/9ad85130c6e2c3636642418fca7c80d9aa339035/docs/source/ko/_toctree.yml#L663)

Reply:
@stevhliu I have completed modifying the toctree, thank you!